package.json

```diff
@@ -1,78 +1,83 @@
-{
```

Comment:
What's this change, line endings?

Reply:
Hmmm, good point, didn't notice that. Let me see if I can clean that up